### DISCUSSION TITLE
VS Code - Formatting

### DISCUSSION THREAD

**Discussion by niklashmbgr** [neuefische/questions#61] [Web] [answered]
hallo zusammen, 

ich probiere eine 'dummyDatabase' anzulegen und diese datei abzuspeichern, jedoch wird bei immer das komma zwischen den objects (die im array liegen) rot unterstrichen und ich bekomme ich im vs code folgende fehlermeldung:
 " Formatting - Extension ‘ESLint’ is configured as formatter but it cannot format ‘JavaScript’-files” 

so sehen die beispieldaten aus:

` export const PRODUCTS = [
	{
	  id: 0,
	  category: "coffee",
	  name: 'Cafe Crema',
	  price: 4.90
	},
	{
	  id: 1,
	  category: "tea",
	  name: "Green Tea",
	  price: 4.90
	}
	,
	{
	  id: 2,
	  category: "coffee",
	  name: "Latte Macchiato",
	  price: 4.90
	}
	,
	{
	  id: 3,
	  category: "tea",
	  name: "Black Tea",
	  price: 4.90
	}
	,
	{
	  id: 4,
	  category: "coffee",
	  name: "Espresso",
	  price: 4.90
	},
	{
	  id: 5,
	  category: "tea",
	  name: "Rooibos Tea",
	  price: 4.90
	},
	{
	  id: 6,
	  category: "tea",
	  name: "Earl Grey Tea",
	  price: 4,90
	}
  ];`

vielen dank vorab!

**Comment by alexclaes** [comment]
Ni @niklashmbgr,

the syntax is broken. Please have a look the price of the last product: `price: 4,90`

The `,` is interpreted as JS syntax. Please use `.` as decimal separator in the price.

Does this solve the issue?

**Reply by niklashmbgr** [reply]
hey @aclaes ,

thanks for the quick answer but the issue is still there... now it looks like this: 
![Screenshot 2022-08-15 at 13 34 23](https://user-images.githubusercontent.com/107849116/184628127-c4a07cf2-4c11-4169-82f6-32da3db11198.png)

**Comment by stefankorder** [comment, marked as answer]
Um den Fehler zu beheben müsst ihr in VS-Code in den Einstellungen "Enables ESLint as formatter" aktivieren:

![grafik](https://user-images.githubusercontent.com/77627144/184630221-b7f1e9c3-716e-4f2e-961d-402c96079479.png)
